### ISSUE TITLE
VoiceOver dropdown announces option items text twice

### ISSUE THREAD

**Issue by Lijing29** [OfficeDev/office-ui-fabric-react#9885]
## Codepen demonstrating the issue
Using this codepen: https://codepen.io/lijinglin29/pen/EqVPpw

## Actual behavior:
When using office-fabric-react Dropdown, VoiceOver announces the options text twice on tabbing to the dropdown and using down arrow to navigate the options.
Browser: Safari

## Expected behavior:
VoiceOver screen reader should announce the option item once when navigating on it.

**Comment by ecraig12345** [comment]
I think the double option reading with the menu open is because we're using the `title` attribute to show the rest of the option text in case it overflows the width of the dropdown. You can see this from my codepen that sets a custom `title` on some of the items: https://codepen.io/ecraig12345/pen/KONMVp?editors=0010

We could potentially render the full text in a TooltipHost instead of using `title`--I think TooltipHost shows on hover but isn't read. But that could cause a significant perf hit (especially for dropdowns with lots of options).

@joschect Do you have any thoughts on this (or know who might) since you're working on the new dropdown? Also @micahgodbolt since we discussed this a bit earlier in the week. Note that this does not occur with Narrator+Edge.

**Comment by ecraig12345** [comment]
Forgot to mention, one potential workaround is that I could make the dropdown respect empty strings passed into `IDropdownOption.title`, so you could essentially turn off the title text.

**Comment by joschect** [comment]
@ecraig12345 Sorry for my slow reply, @cliffkoh might know something about it. I'll take a look too.

@Lijing29 I'm a little confused, is it that text is read when first tabbed into, and again when you press an arrow key? Or is it that when you tab in the text is read twice?

**Comment by Lijing29** [comment]
@joschect Sorry for the late respond. When I use

**Comment by Lijing29** [comment]
Could we reactive this bug as it is not fixed yet!

**Comment by cliffkoh** [comment]
Hi @Lijing29, @ecraig12345 has checked in a workaround that will unblock you on this scenario, if you're optimizing for VoiceOver. The fix should be consumable in less than 3 hours :)

**Comment by Lijing29** [comment]
I have tested it using v7.25.0. If I set the title of the dropdownOption to emprt string, it would read "Apple, text". Here, Apple if the value of the option. Is this what we want? I don`t think we want it read the word "text". I have updated the [codepen](https://codepen.io/lijinglin29/pen/EqVPpw) and you could test it using VoiceOver + Safari.

**Comment by Lijing29** [comment]
@ecraig12345 @cliffkoh Do you have any ideas about that the fix would make the VoiceOver to speak "Apple, text"? Thanks!!

**Comment by cliffkoh** [comment]
Hi @Lijing29, please take a look at https://github.com/OfficeDev/office-ui-fabric-react/wiki/Accessibility-Troubleshooting.

**Comment by Lijing29** [comment]
Hey, @cliffkoh! Thanks for sharing the document. For the part "Supported Configurations: Microsoft Edge & Narrator", does it mean that we would only support Narrator + Edge? 
As this bug is about VoiceOver + Safari, we would not like to fix it. Is it right?

**Comment by cliffkoh** [comment]
Hi @Lijing29, I think it is more accurate to say that we're unable to provide support for it and would recommend opening a bug with the appropriate screen reader vendor and/or browser vendor (our focus is on standards compliance here...).

**Comment by dzearing** [comment]
@cliffkoh One this unclear to me is what exactly VoiceOver is getting tripped up on. Can you provide an isolated repo of something we're doing correctly that VoiceOver doesn't get right?

**Comment by dzearing** [comment]
@ecraig12345 can you please investigate the right fix?

**Comment by ecraig12345** [comment]
I made an [updated codepen](https://codepen.io/ecraig12345/pen/ExYBvEG?editors=0010) with a simple listbox containing button options (like Dropdown uses) and even a plain div option for good measure, and VoiceOver **always** reads "text" after anything with `role="option"` inside a `role="listbox"`. I'm not sure why they chose to do that, but it's not something we can control on the Fabric side.

About the reading of both option text and title, the [aria spec for name computation](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te) seems to imply that screen readers should be choosing **exactly one** of `aria-labelledby`, `aria-label`, `title`, or text content (in that order) for an element's name. So the fact that VoiceOver reads both is a bug on their side. I also noticed that if you decrease VoiceOver's verbosity level (VoiceOver Utility => Verbosity => Speech) to "Low" it stops double-reading the option (though it still appends "text" for some reason).
